doc/Learn/Tutorials/Authentication/HowTo-Cookies.md

```diff
@@ -3,11 +3,88 @@ uid: Learn.Tutorials.Authentication.HowToCookieAuthorization
 
-> [!TIP]
-> This guide assumes you used the Uno.Extensions template to create the solution. Instructions for creating an application from the template can be found [here](xref:Overview.Extensions).
+> [!IMPORTANT]
+> To follow these steps, you first need to have an authentication system set up. We recommend choosing one of the `IAuthenticationProvider` implementations provided by Uno Extensions. Cookie authorization can complement any of the tutorials such as [Get Started with Authentication](xref:Learn.Tutorials.Authentication.HowToAuthentication).
 
 ## Step-by-steps
 
-### 1. 
\ No newline at end of file
+### 1. Enable cookies
+
+- The first step is to opt-in to using cookies. This will allow for writing of returned access and refresh tokens to a cookie, and enables future reading of tokens from the cookie when they are available. You will also be able to name the tokens. Your app should already have an `IHostBuilder` configured to use an authentication provider like below:
+
+    ```csharp
+    protected override void OnLaunched(LaunchActivatedEventArgs args)
+    {
+        var builder = this.CreateBuilder(args)
+            .Configure(host => 
+            {
+                host.UseAuthentication(auth =>
+                    auth.AddCustom(custom =>
+                        custom.Login(
+                            async (sp, dispatcher, tokenCache, credentials, cancellationToken) =>
+                            {
+                                var isValid = credentials.TryGetValue("Username", out var username) && username == "Bob";
+                                return isValid ? 
+                                credentials : default;
+                            })
+                ));
+            });
+    ...
+    ```
+
+- Modify the app to add the `Cookies()` extension method. It configures the `IAuthenticationBuilder` to use cookies by registering a special HTTP request handler that will parse the response for tokens and store them in a cookie. It will apply them to future requests.
+
+    ```csharp
+    protected override void OnLaunched(LaunchActivatedEventArgs args)
+    {
+        var builder = this.CreateBuilder(args)
+            .Configure(host => 
+            {
+                host.UseAuthentication(auth =>
+                    auth.AddCustom(custom =>
+                        custom.Login(
+                            async (sp, dispatcher, tokenCache, credentials, cancellationToken) =>
+                            {
+                                var isValid = credentials.TryGetValue("Username", out var username) && username == "Bob";
+                                return isValid ? 
+                                credentials : default;
+                            })
+                ), 
+                configureAuthorization: builder =>
+                {
+                    builder
+                        .Cookies(/* options */);
```

Comment:
Can you make a note here what the default authorization is - I thought it was Cookies already but you'll need to check this.